libraries/AP_Airspeed/AP_Airspeed_ASP5033.h

```diff
@@ -0,0 +1,49 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+/*
+  backend driver for airspeed from I2C
+ */
+#include "AP_Airspeed_Backend.h"
+
+class AP_Airspeed_ASP5033 : public AP_Airspeed_Backend
+{
+public:
+    AP_Airspeed_ASP5033(AP_Airspeed &frontend, uint8_t _instance);
+    ~AP_Airspeed_ASP5033(void) {}
+    
+    // probe and initialise the sensor
+    bool init() override;
+
+    // return the current differential_pressure in Pascal
+    bool get_differential_pressure(float &pressure) override;
+
+    // return the current temperature in degrees C, if available
+    bool get_temperature(float &temperature) override;
+
+private:
+    void timer();
+    int32_t pressraw;
```

Comment:
This should be a local to the `timer` function.

Reply:
done

Reply:
This wasn't done.